File: src/test/java/br/com/model/classes/NovoProdutoFornecedorTest.java
package br.com.model.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.classes.NovoProdutoFornecedor;

public class NovoProdutoFornecedorTest {
    @Test
    void testNovaListaProdutosFornecedor() {
        System.out.println(NovoProdutoFornecedor.novaListaProdutosFornecedor().toString());
    }
}

File: src/test/java/br/com/model/NovoProdutoFornecedorTest.kt
package br.com.model

import br.com.newObjects.model.NovoProdutoFornecedor
import org.junit.jupiter.api.Test

class NovoProdutoFornecedorTest {
    @Test
    fun testNovaListaProdutosFornecedor() {
        println(NovoProdutoFornecedor.novaListaProdutosFornecedor().toString())
    }
}
